### DISCUSSION TITLE
Linking `ncdata` and `data`

### DISCUSSION THREAD

**Discussion by J-Kappes** [szaimen/Nextcloud-NAS-Guide#124] [Q&A] [answered]
I don't see how to get the external SSD to show up in the NC web UI. I moved away a user's `files/` under `/mnt/ncdata` and replaced it with a symlink to that user's folder on the external SSD `/mnt/data` but I keep seeing the same default contents on that user's web UI.

**Comment by szaimen** [comment]
Hi, please ask this at https://help.nextcloud.com

**Reply by J-Kappes** [reply]
But isn't this one of the goals of your guide: Storing one's nextcloud files on the external SSD? That's why the sensible folder structure for the external SSD includes directories named "Sync".

**Reply by szaimen** [reply, marked as answer]
> But isn't this one of the goals of your guide: Storing one's nextcloud files on the external SSD? That's why the sensible folder structure for the external SSD includes directories named "Sync".

No, the idea is that ncdata stays on the internal SSD that also contains the OS. Then an additional is getting mounted during https://szaimen.github.io/Nextcloud-NAS-Guide/docs/data-ssd and then later on used for your data via https://szaimen.github.io/Nextcloud-NAS-Guide/docs/create-shares
